src/component/NavigationBar.jsx

```diff
@@ -1,15 +1,13 @@
-import { Link } from "react-router-dom"
+import { Link } from "react-router-dom";
 
 const NavigationBar = () => {
+  return (
+    <section className="NavigationBar">
+      <Link to="">Home</Link>|<Link to="/topics/cooking">Cooking</Link>|
+      <Link to="/topics/coding">Coding</Link>|
+      <Link to="/topics/football">Football</Link>
+    </section>
+  );
+};
```

Comment:
You should have an API endpoint that serves the topics

It would be good to use this instead of hard coding the topics in by hand!